dp/pipeline/epc_packet_framework.h

```diff
@@ -45,6 +45,9 @@ extern uint64_t num_dns_processed;
 /**
  * S1U port id.
  */
+#ifdef USE_AF_PACKET
+#define	S1U_PORT_VETH_ID	S1U_PORT_ID + NUM_SPGW_PORTS
```

Comment:
We are making a coupling here.
Current code itself seems to hardcode the port IDs of S1U and SGI. Not sure if it would work if we do not have control of the order. Can you open an issue if it stops working when order is reversed, both for current code and for this patch?

Ideally we would use something like these - app.sgi_port, app.s1u_port which would capture at run-time what port IDs they came up as.

Reply:
I'm currently hoping that this pairing sticks. Swapping the `S1U_PORT_ID` and `SGI_PORT_ID` values messes up original dp's initialization routine as well. This issue should be tackled separately. I can come back to this thereafter.

Reply:
Please open an issue to address this limitation in the original code. We should also update release notes if its not captured already.